server/service/src/programs/patient/patient_updated.rs

```diff
@@ -117,6 +126,7 @@ pub fn update_patient_row(
             .and_then(|n| n.created_datetime.clone())
             .or(Some(update_timestamp.naive_utc())), // assume there is no earlier doc version
         is_deceased: patient.is_deceased.unwrap_or(false),
+        date_of_death,
         national_health_number: code_2,
```

Comment:
Currently, its only possible to set the date of death through a patient document...